config/initializers/assets.rb

```diff
@@ -12,4 +12,14 @@
 # application.js, application.css, and all non-JS/CSS in the app/assets
 # folder are already added.
 # Rails.application.config.assets.precompile += %w( admin.js admin.css )
-Rails.application.config.assets.precompile += %w[main.tablet.css main.desktop.css]
+require 'find'
+# Auto precompile Javascripts
+js_path = File.join(Rails.root, 'app/assets/javascripts/')
+Find.find(js_path).each do |file|
+  Rails.application.config.assets.precompile << $1.sub(js_path, '') if file.match(/^(.*\.js)(\.coffee)?(\.erb)?$/)
```

Comment:
__[RuboCop]__

Performance/RedundantMatch: Use `=~` in places where the `MatchData` returned by `#match` will not be used.


<!-- Comment by [SideCI](https://sideci.com) -->